src/test/rpc/AccountObjects_test.cpp

```diff
@@ -316,10 +316,215 @@ class AccountObjects_test : public beast::unit_test::suite
         }
     }
 
+    void testObjectTypes()
+    {
+        testcase ("object types");
```

Comment:
Looks like (elsewhere in the file) there usually isn't a space between `testcase` and `(`?

i.e.
```
testcase("object types");
```